torch/_torch_docs.py

```diff
@@ -6788,6 +6788,11 @@ def merge_dicts(*dicts):
 When :attr:`other` is a tensor, the shapes of :attr:`input` and
 :attr:`other` must be :ref:`broadcastable <broadcasting-semantics>`.
 
+Note:
+    Complex inputs are not supported. In some cases, it is not mathematically
+    possible to satisfy the definition of a modulo operation with complex numbers.
+    See `Github issue <https://github.com/pytorch/pytorch/issues/34266>`_ for more information.
```

Comment:
Let's just kill this last line; I'm not sure we want to cite issues in the docs.